src/components/questionnaire/new-questionnaire-card.tsx

```diff
@@ -17,21 +17,15 @@ import strings from "src/localization/strings";
  * Interface for the NewQuestionnaireCard component
  */
 interface Props {
-  handleAddQuestionSubmit: (
-    questionText: string,
-    options: { label: string; value: boolean }[]
-  ) => void;
+  handleAddQuestionSubmit: (question: string, options: { label: string; value: boolean }[]) => void;
 }
 
 /**
  * New Questionnaire Card Component
  */
-const NewQuestionnaireCard = ({handleAddQuestionSubmit}: Props) => {
-  
-  const [questionText, setQuestionText] = useState("");
-  const [options, setOptions] = useState([
-    { label: "", value: false }
-  ]);
+const NewQuestionnaireCard = ({ handleAddQuestionSubmit }: Props) => {
+  const [question, setQuestionText] = useState("");
+  const [options, setOptions] = useState([{ label: "", value: false }]);
```

Comment:
Change typing. Would like to keep this default value, so it would have one empty QuestionOption available at first. If there is better way to handle this, let me know.